mocks/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
```

Comment:
`mocks` module is where all test mocks live.
It is `sharedTest` directory in v1, but it now shows a warning below and does not work correctly in Android Studio, so I created a dedicated module.

```
Duplicate content roots detected: Path [/Users/yshrsmz/repos/github.com/yshrsmz/bucketeer-android-sdk/bucketeer/src/sharedTest/java] of module [bucketeer-android-sdk.bucketeer.unitTest] was removed from modules [bucketeer-android-sdk.bucketeer.androidTest]
```